.github/workflows/main.yml

```diff
@@ -36,3 +36,29 @@ jobs:
       - uses: hadolint/hadolint-action@v1.5.0
         with:
           dockerfile: Dockerfile
+  build_image:
+    needs: build
+    if: github.event_name == 'push'
```

Comment:
Can we make sure it is on the `master` branch?

Reply:
all jobs use global settings set in the top of the file